README.rst

```diff
@@ -26,8 +26,8 @@ Website: https://fairlearn.org/
 Current release
 ---------------
 
--  The current stable release is available at
-   `Fairlearn v0.6.2 <https://github.com/fairlearn/fairlearn/tree/release/v0.6.2>`__.
+-  The current stable release is available on
+   `PyPI <https://badge.fury.io/py/fairlearn>`__.
```

Comment:
How standard / accepted is `badge.fury.io`? I'd be more comfortable linking to https://pypi.org/project/fairlearn/

Reply:
Somehow I wasn't able to get a pypi link pointing at the latest version, but yours does! Not sure how I didn't find that one 🙂 I'm replacing it in the next commit.